pkg/util/errutil/errors.go

```diff
@@ -41,6 +41,76 @@ func NewBase(reason StatusReason, msgID string, opts ...BaseOpt) Base {
 	return b
 }
 
+// NotFound initializes a new [Base] error with reason StatusNotFound.
+//
+// See NewBase for more information.
+func NotFound(msgID string, opts ...BaseOpt) Base {
+	return NewBase(StatusNotFound, msgID, opts...)
+}
+
+// BadRequest initializes a new [Base] error with reason StatusBadRequest.
+//
+// See NewBase for more information.
+func BadRequest(msgID string, opts ...BaseOpt) Base {
+	return NewBase(StatusBadRequest, msgID, opts...)
+}
+
+// ValidationFailed initializes a new [Base] error with reason StatusValidationFailed.
+//
+// See NewBase for more information.
+func ValidationFailed(msgID string, opts ...BaseOpt) Base {
+	return NewBase(StatusValidationFailed, msgID, opts...)
+}
+
+// Internal initializes a new [Base] error with reason StatusInternal.
+//
+// See NewBase for more information.
+func Internal(msgID string, opts ...BaseOpt) Base {
+	return NewBase(StatusInternal, msgID, opts...)
+}
+
+// Unknown initializes a new [Base] error with reason StatusUnknown.
+//
+// See NewBase for more information.
+func Unknown(msgID string, opts ...BaseOpt) Base {
+	return NewBase(StatusUnknown, msgID, opts...)
+}
+
+// Unknown initializes a new [Base] error with reason StatusUnknown.
```

Comment:
```suggestion
// Unknown initializes a new [Base] error with reason StatusTimeout.
```

Reply:
Nice spotted. Managed to miss the function name as well :) 

```suggestion
// Timeout initializes a new [Base] error with reason StatusTimeout.
```